utilities/pipelines/staticValidation/module.tests.ps1

```diff
@@ -1198,7 +1198,14 @@ Describe 'Deployment template tests' -Tag Template {
 Describe "API version tests [All apiVersions in the template should be 'recent']" -Tag ApiCheck {
 
     $testCases = @()
-    $ApiVersions = Get-AzResourceProvider -ListAvailable
+    $apiSpecsFilePath = Join-Path $repoRootPath 'utilities' 'src' 'apiSpecsList.json'
+
+    if (-not (Test-Path $apiSpecsFilePath)) {
+        Write-Verbose "Skipping API tests as no API version are available in path [$apiSpecsFilePath]"
```

Comment:
This means that users not willing to test api versions can simply delete the file from the repo? If that is considered as an option, we should document it.

Reply:
added